src/views/recipeView.js

```diff
@@ -104,7 +105,35 @@ ${recipe.ingredients
 </div>`;
   recipeContainer.innerHTML = '';
   recipeContainer.insertAdjacentHTML('afterbegin', html);
+  const btnIncrease = document.querySelector('.btn--increase-servings');
+  const btnDecrease = document.querySelector('.btn--decrease-servings');
+  btnIncrease.addEventListener('click', btnIncreaseHandler);
+  btnDecrease.addEventListener('click', btnDecreaseHandler);
+  btnDecrease.disabled = servings === 1 ? true : false;
+
+  function btnIncreaseHandler() {
+    servings = servings + 1;
+
+    newQuantity();
+    recipe.servings = servings;
+    renderRecipe(recipe);
+  }
+  function btnDecreaseHandler() {
+    servings = servings - 1;
+
+    newQuantity();
+    recipe.servings = servings;
+    renderRecipe(recipe);
+  }
+  function newQuantity() {
+    recipe.ingredients.forEach(ing => {
+      ing.quantity = (ing.quantity * servings) / recipe.servings;
+    });
+    // console.log(servings, 'kjhg');
+    // console.log(recipe.ingredients, 'jhg');
+  }
```

Comment:
why u put this inside another function ? can move outside of main function ?

Reply:
okay